### PR TITLE
[docs] Updates to voyager known issues

### PR DESCRIPTION
**Pull request by aishwarya24**
Added new issues and modified some old ones.

@netlify /preview/migrate/known-issues/mysql/

### CODE REVIEW

File: docs/content/preview/migrate/known-issues/oracle.md [hunk 1]
```diff
@@ -243,3 +244,33 @@ CREATE TABLE employees (
...
 ) PARTITION BY RANGE (hire_date) ;
 ```
+
+---
+
+### Negative scale is not supported
+
+**GitHub**: [Issue #779](https://github.com/yugabyte/yb-voyager/issues/779)
+
+**Description**: Oracle supports negative scale where you can round up the values to the power of tens corresponding to the scale provided. Negative scale support is not supported in PostgreSQL and therefore in YugabyteDB.
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues/oracle.md:254]: 1. The values are round down. 
2. Can we please reduce the redundancy by removing support from a place.
`Negative scale support is not supported`
Thank You

File: docs/content/preview/migrate/known-issues/mysql.md [hunk 2]
```diff
@@ -267,28 +267,29 @@ CREATE OR REPLACE VIEW v1 AS SELECT foo(bar.id::int) AS p_name FROM bar;
...
-CREATE PROCEDURE foo(p_id int)
-deterministic
+CREATE FUNCTION func (p_id int)
+RETURNS VARCHAR(20)
+READS SQL DATA
   BEGIN
     DROP TEMPORARY TABLE IF EXISTS temp;
     CREATE TEMPORARY TABLE temp(id int, name text);
     INSERT INTO temp(id,name) SELECT id,p_name FROM bar WHERE p_id=id;
     SELECT name FROM temp;
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues/mysql.md:279]: Please put the select statement in a return clause.
`return (SELECT name FROM temp);`

File: docs/content/preview/migrate/known-issues/postgresql.md [hunk 3]
```diff
@@ -59,3 +61,79 @@ CREATE TABLE public.sales_region (
...
+
+---
+
+### Index creation on partitions fail for some YugabyteDB builds
+
+**GitHub**: [Issue 14529](https://github.com/yugabyte/yugabyte-db/issues/14529)
+
+**Description**: If you have a partitioned table with indexes on it, the migration will fail with an error for YugabyteDB `2.15` or `2.16` due to a regression.
+
+Note that it is fixed in the release [12.17.1.0](../../../releases/release-notes/v2.17/#v2.17.1.0).
```
Comment (ddhodge) [inline docs/content/preview/migrate/known-issues/postgresql.md:73]:
```suggestion
Note that this is fixed in release [12.17.1.0](../../../releases/release-notes/v2.17/#v2.17.1.0).
```

File: docs/content/preview/migrate/known-issues/postgresql.md [hunk 4]
```diff
@@ -59,3 +61,79 @@ CREATE TABLE public.sales_region (
...
+
+```sql
+CREATE TABLE foo(n1 int PRIMARY KEY, n2 int);
+CREATE VIEW v1 AS
+  SELECT n1,n2
+  FROM foo
+  GROUP BY n1;
+```
+
+The exported schema for `view.sql` is as follows:
```
Comment (ddhodge) [inline docs/content/preview/migrate/known-issues/postgresql.md:123]: Should this be rule.sql? they are both view.sql currently

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/migrate/known-issues/postgresql.md (hunk 4 above)
Comment (aishwarya24) [reply docs/content/preview/migrate/known-issues/postgresql.md:123]: yes , thanks

File: docs/content/preview/migrate/known-issues/mysql.md [hunk 5]
```diff
@@ -304,7 +305,7 @@ CREATE OR REPLACE PROCEDURE foo (p_id integer) AS $body$
     DROP TABLE IF EXISTS temp;
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues/mysql.md:305]: Please change the create DDL from procedure to function.

```
CREATE OR REPLACE FUNCTION func (p_id integer) RETURNS varchar AS $body$
BEGIN
```

**Review by shubham-yb — APPROVED** [review]

File: docs/content/preview/migrate/known-issues/oracle.md [hunk 6]
```diff
@@ -243,3 +221,86 @@ CREATE TABLE employees (
...
+
+- Modify the DDL by removing the schema name `test` from the DDL:
+
+    ```sql
+    CREATE OR REPLACE VIEW offices AS SELECT * FROM locations;
+    ```
+
+---
+
+### Small-sized CLOB/NCLOB data is only supported
```
Comment (ddhodge) [inline docs/content/preview/migrate/known-issues/oracle.md:300]:
```suggestion
### Large-sized CLOB/NCLOB data is not supported
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**